src/bicep/examples/newWorkload/newWorkload.bicep

```diff
@@ -105,7 +163,7 @@ module workloadVirtualNetworkPeerings '../../modules/spokeNetworkPeering.bicep'
 
 module hubToWorkloadVirtualNetworkPeering './modules/hubNetworkPeering.bicep' = {
   scope: subscription(hubSubscriptionId)
-  name: 'hubToWorkloadVirtualNetworkPeering'
+  name: take('hub-to-${workloadName}-vnet-peering', 64)
```

Comment:
I updated this to match the convention used in `hubNetworkPeerings.bicep`.  Let me know if these are changes are correct by resolving this conversation.

https://github.com/Azure/missionlz/blob/199b986c770d0d331b6d99fce39d9b9807af2e28/src/bicep/modules/hubNetworkPeerings.bicep#L5

Reply:
thanks for the update